# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: editor/txmgr/TransactionManager.cpp
# Commit: b9ebe58001d7
# Full Hash: b9ebe58001d787bb0bbad6d39ae8681966d5a77f
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2020-04-04 09:29:13
# Regressor Bug: 1619914
# File Overlap Count: 1
# Description:
#   Bug 1619914 - part 2: Mark transaction class methods and their caller methods as `MOZ_CAN_RUN_SCRIPT r=m_kato
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D69303
# ==============================================================================

diff -r ce2087ddab5c -r b9ebe58001d7 editor/txmgr/TransactionManager.cpp
--- a/editor/txmgr/TransactionManager.cpp	Fri Apr 03 08:30:37 2020 +0000
+++ b/editor/txmgr/TransactionManager.cpp	Fri Apr 03 08:32:01 2020 +0000
@@ -211,7 +211,7 @@
     return NS_OK;
   }
 
-  rv = BeginTransaction(0, aData);
+  rv = BeginTransaction(nullptr, aData);
   NS_WARNING_ASSERTION(NS_SUCCEEDED(rv),
                        "TransactionManager::BeginTransaction() failed");
 